tests/cupy_tests/statistics_tests/test_order.py

```diff
@@ -6,7 +6,7 @@
 
 import cupy
 import cupy.core._accelerator as _acc
-from cupy import testing
+from cupy import cuda, testing
```

Comment:
Would you split this line to import a module per line to follow our guideline that requires it in general?
```suggestion
from cupy import cuda
from cupy import testing
```